src/main/java/uk/gov/hmcts/darts/common/repository/TranscriptionDocumentRepository.java

```diff
@@ -46,7 +46,7 @@ public interface TranscriptionDocumentRepository extends JpaRepository<Transcrip
              ((cast(:requestedAtFrom as TIMESTAMP)) IS NULL OR (t.createdDateTime >= :requestedAtFrom)) AND
              (:owner IS NULL OR (wfa.userFullName ILIKE CONCAT('%', cast(:owner as text), '%'))) AND
              ((cast(:requestedAtTo as TIMESTAMP)) IS NULL OR t.createdDateTime <= :requestedAtTo)
-             AND t.isCurrent = true        
+             AND t.isCurrent = true and (tmd.isHidden = false or tmd.isHidden = :allowHidden)
```

Comment:
Quick formatting suggestion

```suggestion
             AND t.isCurrent = true
             AND (tmd.isHidden = false OR tmd.isHidden = :allowHidden)
```

Reply:
Agreed :)